DotNet/RoadmapLogic/RoadmapImage.cs

```diff
@@ -73,16 +79,18 @@ public static MemoryStream MakeImage(Input input, Settings settings)
 
         private static void DrawLegsAndPlacards(Input input, Settings settings, Fonts fonts, Image<Rgba32> image, IEnumerable<Quarter> quarters)
         {
-            string missingProjectMessageText = "Alert! Project(s) have been omitted from the roadmap. Please Review.";
+            var noProjectWithinQuartersMessage = string.Empty;
+
             RendererOptions renderOptions = new RendererOptions(fonts.PlacardFont);
-            bool hasMissingProjects = false;
+            var hasMissingProjects = false;
+            var hasProjectBeforeStartQuarter = false;
+            var hasProjectAfterLastQuarter = false;
 
             try
             {
                 if (!input.Projects.Any(p => quarters.Any(q => q.Equals(Quarter.GetQuarter(p.Date)))))
                 {
-                    missingProjectMessageText = $"Alert! No Project is within start date and '{quarters.Count()}' quarters to report. Please Review.";
-                    hasMissingProjects = true;
+                    noProjectWithinQuartersMessage = $"Alert! No Project is within start date and '{quarters.Count()}' quarters to report. Please Review.";
```

Comment:
This message could be cleaned up too and perhaps stored as a const with the others. I don't know that we need to show the user "Alert!" either.

Reply:
Since this message is including the number of quarter's so it cannot be stored as constant; however, if we don't want to include number of quarters, then it could be stored as constant like other methods.

Reply:
Oh true, didn't realize that.